### ISSUE TITLE
Attach gdb to pid

### ISSUE THREAD

**Issue by christian-schmaler-snkeos** [RustDT/RustDT#153]
I do not find a way to attach the debugger to an already running process with a certain pid. It would be great if that option could be added in the future. Is there a workaround for the meantime?

**Comment by bruno-medeiros** [comment]
Yeah, it would be nice to support that, but no plans to, at the moment.

Hum, as a workaround you could try using CDT's "C/C++ Attach to Application" launch. RustDT's debugging integration is only a thin wrap around CDT, with just a few minor additions. But it might be possible to use "C/C++ Attach to Application" directly (there is a field in the launch configuration to specify a C project, but you can leave it blank, it's not required).

I haven't tried it myself as I'm not familiar with how GDB's attach works (does it even work on Windows?). But I tried the "C/C++ Application" launch (not the attach, just the regular one) on a Rust binary and it works. The difference is that pretty printers may not be configured automatically, and also the Rust stdlib source path may not be configured automatically (although I've just noticed, there seems to be bug here even when using the "Rust Application" launch, in the latest RustDT version)
